packages/react-components/react-components/src/AccessibilityScenarios/Input.stories.tsx

```diff
@@ -108,20 +130,18 @@ const RegistrationFormInputsAccessibility = () => {
   const formValidation = useFormValidation(handleSubmit);
 
   const [isPasswordVisible, setIsPasswordVisible] = React.useState(false);
+  const [isSendNewsletter, setIsSendNewsletter] = React.useState(false);
   const [isSubmittedAndValid, setIsSubmittedAndValid] = React.useState(false);
 
   React.useEffect(() => {
     // If the form is submitted and has errors, focus the first error fiel, otherwise do nothing
-    if (!formState.isSubmitting || formState.isValid) {
+    if (!formState.isSubmitted || formState.isValid) {
       return;
     }
     const firstErrorName = Object.keys(errors)[0] as keyof FormInputs;
     const firstErrorField = document.getElementById(firstErrorName);
-
-    setTimeout(() => formValidation.notifyFormFieldError(firstErrorName), 200);
-
     if (firstErrorField) {
-      firstErrorField.focus();
+      setTimeout(() => firstErrorField.focus(), 500);
```

Comment:
what's the purpose of the half-second timeout?

Reply:
I cannot recall the reason for it now. However, I have removed it and looks it works okay :-).